src/main/java/seedu/address/logic/commands/AddTaskCommand.java

```diff
@@ -25,23 +25,25 @@
 public class AddTaskCommand extends Command {
 
     /* Message printed if wrong usage */
-    public static final String MESSAGE_USAGE = "addt" + ": Adds a task to NUS Classes. "
-            + "Parameters: "
+    public static final String COMMAND_WORD = "addt";
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Adds a task to NUS Classes.\n"
+            + "Usage: "
+            + COMMAND_WORD + " "
             + PREFIX_TASKNAME + "TASKNAME "
             + PREFIX_DATETIME + "DATETIME [, ENDDATETIME]"
-            + " [" + PREFIX_TAG + "TAG] "
-            + " [" + PREFIX_LINK + "LINK] "
-            + " [" + PREFIX_RECURRING + "PERIOD RECURRENCE]\n"
+            + " [" + PREFIX_TAG + "TAG" + "] "
+            + " [" + PREFIX_LINK + "LINK" + "] "
+            + " [" + PREFIX_RECURRING + "PERIOD RECURRENCE" + "] \n"
```

Comment:
Why these changes?

Reply:
Yeah true, not really necessary. Changing them back